src/include/platform/PersistedStorage.h

```diff
@@ -87,6 +92,147 @@ CHIP_ERROR Read(Key aKey, uint32_t & aValue);
  */
 CHIP_ERROR Write(Key aKey, uint32_t aValue);
 
+class KeyValueStore
+{
+public:
+    virtual ~KeyValueStore() = default;
+    /**
+     * @brief
+     *   Reads the value of an entry in the KVS. The value is read into the
+     *   provided buffer and the number of bytes read is returned. If desired,
+     *   the read can be started at an offset.
+     *
+     *   If the output buffer is too small for the value, Get returns
+     *   CHIP_ERROR_BUFFER_TOO_SMALL with the number of bytes read returned in
+     *   read_bytes_size, which should be the buffer_size.
+     *
+     *   The remainder of the  value can be read by calling get with an offset.
+     *
+     * @param[in]     key               The name of the key to get, this is a
+     *                                  NULL terminated string.
```

Comment:
super nit: null-terminated string.

null is a word. NULL is a particular pointer value.

Reply:
Done